src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3RequestStream.cs

```diff
@@ -275,6 +275,11 @@ public async Task<HttpResponseMessage> SendAsync(CancellationToken cancellationT
                 Exception abortException = _connection.Abort(HttpProtocolException.CreateHttp3ConnectionException(code, SR.net_http_http3_connection_close));
                 throw new HttpRequestException(HttpRequestError.HttpProtocolError, SR.net_http_client_execution_error, abortException);
             }
+            catch (QuicException ex) when (ex.QuicError == QuicError.OperationAborted && cancellationToken.IsCancellationRequested)
+            {
+                OperationCanceledException wrapperOce = new OperationCanceledException(ex.Message, ex, cancellationToken);
+                throw new HttpRequestException(HttpRequestError.Unknown, SR.net_http_client_execution_error, wrapperOce);
```

Comment:
You should be doing something like this:
https://github.com/dotnet/runtime/blob/d22909e0225e885b85c55b9242fb51c81e5256ad/src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3RequestStream.cs#L294

The exception going out should be `TaskCancelledException`.

Reply:
I thought we should be throwing OCE, because #98220 explicitly states that. What is the reason for throwing `TaskCancelledException`?

Reply:
`TaskCancelledException` derives from `OperationCancelledException`